lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1477,10 +1488,21 @@ void SwiftASTContext::AddExtraClangArgs(std::vector<std::string> ExtraArgs) {
     if (clang_argument.startswith("-Werror")) {
       continue;
     }
+
+    if (clang_argument.empty()) {
+      continue;
+    }
+
     // Otherwise add the argument to the list.
-    if (!is_macro)
+    if (!IsMacroDefinition(clang_argument))
       ApplyWorkingDir(clang_argument, cur_working_dir);
-    AddClangArgument(clang_argument.str(), unique);
+
+    bool unique = ShouldUnique(clang_argument);
+    auto clang_argument_str = clang_argument.str();
+    if (!unique || unique_flags.count(clang_argument_str)) {
```

Comment:
`if (!ShouldUnique(...) || `

Reply:
It gets out of 80 symbols restriction. Do you mean to split condition in two lines?

Reply:
I managed it by shortening clang_argument_str :)